public/docs/_examples/toh-6/ts/app/app.module.ts

```diff
@@ -12,7 +12,7 @@ import { HttpModule }    from '@angular/http';
 
 // #enddocregion v1
 // Imports for loading & configuring the in-memory web api
-import { InMemoryWebApiModule } from 'angular-in-memory-web-api';
+import { InMemoryWebApiModule } from 'angular-in-memory-web-api/in-memory-web-api.module';
```

Comment:
Is there a reason we have to import this way instead of just importing from `angular-in-memory-web-api`?